inc/admin/class-assets.php

```diff
@@ -77,7 +77,7 @@ protected function prepareAdminData() {
 		$hasToken      = ! empty( $token );
 		$hasCustomerId = ! empty( $customerData ) && ! empty( $customerData['customer_id'] );
 		$showCTBs      = $hasToken && $hasCustomerId;
-		$isJarvis      = get_option( 'bh_platform' ) === 'jarvis' ? 'true' : null;
+		$isJarvis      = bh_is_jarvis() ? 'true' : null;
```

Comment:
Also, if we're already setting this in the plugin container, then let's just use that value instead of running the file check again. 

Perhaps the `bh_is_jarvis()` function should check the container first to see if it's already been stored and then push it in if it isn't there. That would avoid a filesystem call entirely and pull the value out of the active PHP request memory when available.

Reply:
We could move it down in the prepareData method so that it's only loaded on the Bluehost plugin pages. I don't think we need it globally in admin? Or does it change the ctb availability?